addons/number-pad/addon.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "Always show number pad",
-  "description": "Show Scratch's number pad input when editing number fields on all devices, instead of only touchscreen devices.",
+  "description": "Show Scratch's virtual number pad when editing number fields in blocks on all devices, instead of only touchscreen devices.",
   "credits": [
     {
       "name": "GarboMuffin"
     }
   ],
   "info": [
     {
-      "text": "A number pad will show when editing number inputs from certain blocks, such as \"set x to\".",
+      "text": "A number pad will show when editing number inputs from blocks such as \"move () steps\".",
```

Comment:
Is this note necessary?

Reply:
> Is this note necessary?

See #5906.

Reply:
What do you want us to do with it? Do you think it should be part of the description? Reworded? Removed entirely?